src/screens/NewLocationScreen.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+
+import * as React from 'react';
+
+import {
+  Body,
+  Button,
+  Container,
+  Content,
+  Header,
+  Icon,
+  Left,
+  Title,
+} from 'native-base';
+
+type Props = {|
+  navigation: Object,
```

Comment:
Can you add a todo to add better typing here?

Reply:
added in on of the places.